### PR TITLE
Added Ret Cloak. Fixed minor bug with Crusaders Strike

### PR DESCRIPTION
**Pull request by CollCrom**

### CODE REVIEW

File: src/Parser/DestructionWarlock/Modules/Items/Legendaries/OdrShawlOfTheYmirjar.js [hunk 1]
```diff
@@ -19,7 +19,7 @@ const AFFECTED_SPELLS = [
   SPELLS.IMMOLATE_DEBUFF.id,
   SPELLS.IMMOLATE_CAST.id,
   // don't have these two statistically proven to be buffed, might be just RNG, but looks like they are
-  SPELLS.MAGISTRIKE_RESTRAINTS_CHAOS_BOLT.id,
+  //SPELLS.MAGISTRIKE_RESTRAINTS_CHAOS_BOLT.id,
```
Comment (rp4rk) [inline src/Parser/DestructionWarlock/Modules/Items/Legendaries/OdrShawlOfTheYmirjar.js]: Can this please be uncommented, as far as I can tell this is out of the scope of this PR?

Comment (sMteX) [reply src/Parser/DestructionWarlock/Modules/Items/Legendaries/OdrShawlOfTheYmirjar.js]: We dealt with this on Discord late night - seemed like a PFM issue on his localhost, after restarting server/IDE it worked fine.

**Comment by CollCrom** [comment]
yeah some weird bug. Fixed it though.

**Comment by anom0ly** [comment]
Looks good now. Thanks!

**Pull request merged**